impeller/playground/backend/metal/playground_impl_mtl.mm

```diff
@@ -63,16 +63,18 @@
 PlaygroundImplMTL::PlaygroundImplMTL(PlaygroundSwitches switches)
     : PlaygroundImpl(switches),
       handle_(nullptr, &DestroyWindowHandle),
-      data_(std::make_unique<Data>()) {
+      data_(std::make_unique<Data>()),
+      concurrent_loop_(fml::ConcurrentMessageLoop::Create()) {
```

Comment:
This is more or less duplicating the problem that the VKContext has wherein we should only be creating a single concurrent message loop once per engine.

Reply:
@jason-simmons @gaaclarke fyi

Reply:
FWIW We switched around ContextVK so that it is made to support sharing a concurrent message loop.  There's no known problem.  This came from Chinmay's design goal of wanting to share these.  I'm not sure what that means for your PR, I'm just clarifying that point.

Reply:
ContextVK is still creating its own concurrent message loop? 

https://github.com/flutter/engine/blob/0ae3719d70432130594c4838c36d5849ae6a01ff/shell/platform/android/android_context_vulkan_impeller.cc#L44

Reply:
`ContextVK` takes in a shared_ptr to a task queue which can be shared across multiple engines, but currently is not: https://github.com/flutter/engine/blob/0ae3719d70432130594c4838c36d5849ae6a01ff/impeller/renderer/backend/vulkan/context_vk.h#L43

I had a PR out that made ContextVK own a single concurrent message loop but we decided we didn't want to shut the door on sharing concurrent message loops between engines.  I haven't read this PR through, I'm just responding to the comment "the problem that the VKContext has wherein we should only be creating a single concurrent message loop once per engine".  There is no problem that I know of with Vulkan and we made sure of that with our recent work.  Let me know if you want me review this PR if you think it would be helpful.